### ISSUE TITLE
没歌词时显示上一首歌曲的歌曲

### ISSUE THREAD

**Issue by GoogleCodeExporter** [memeco/osd-lyrics#274]
```
What steps will reproduce the problem?
请列出导致问题的操作
1.先播放有歌词的歌曲
2.切换到没有歌词的歌曲
3.进行Scrolling mode<->OSD mode之间切换


What's the output in terminal (run with command `osd-lyrics -d debug')?
请给出在终端下运行的输出（使用“osd-lyrics -d
debug”来运行）

What's the player and which version you are using?
您使用的播放器是什么？它的版本号是多少？

What's version of OSD Lyrics?
OSD Lyrics的版本是多少？
0.4.3

What's your Linux distro?
您使用哪种发行版？
Ubuntu 12.04 x86_64 en_US.UTF-8

```

Original issue reported on code.google.com by `weitianl...@gmail.com` on 6 Jul 2012 at 2:28

**Comment by GoogleCodeExporter** [comment]
Original comment by `tigerso...@gmail.com` on 27 Jul 2012 at 2:48
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
fixed in commit 6a494214
```

Original comment by `tigerso...@gmail.com` on 27 Jul 2012 at 3:13
- Changed state: **Fixed**
